### PR TITLE
Add support of markup for StaticText

### PR DESCRIPTION
**Pull request by eldarkg**

### CODE REVIEW

File: output/plugins/common/xml/common.cppcode [hunk 1]
```diff
@@ -86,8 +86,9 @@ Written by
   <templates class="wxStaticText">
     <template name="declaration">#class* $name;</template>
     <template name="include">@#include &lt;wx/stattext.h&gt;</template>
-    <template name="construction">$name = new #class( #wxparent $name, $id, $label, $pos, $size, $style #ifnotnull $window_style @{ |$window_style @} #ifnotnull $window_name @{, $window_name @} );</template>
-  	<template name="settings">$name->Wrap( $wrap );</template>
+    <template name="construction">$name = new #class( #wxparent $name, $id, wxEmptyString, $pos, $size, $style #ifnotnull $window_style @{ |$window_style @} #ifnotnull $window_name @{, $window_name @} );</template>
```
Comment (jhasse) [inline output/plugins/common/xml/common.cppcode]: To reduce the code needed when not using markup, I would still pass the label to the constructor here, when `$markup` doesn't equal 1. Ideally the output shouldn't change at all when updating wxFormBuilder without changing a setting.

Comment (eldarkg) [reply output/plugins/common/xml/common.cppcode]: OK

File: plugins/common/common.cpp [hunk 2]
```diff
@@ -368,14 +368,20 @@ class StaticTextComponent : public ComponentBase
...
-			obj->GetPropertyAsString(_("label")),
-			obj->GetPropertyAsPoint(_("pos")),
-			obj->GetPropertyAsSize(_("size")),
-			obj->GetPropertyAsInteger(_("style")) | obj->GetPropertyAsInteger(_("window_style")));
+		wxString label = obj->GetPropertyAsString( _("label") );
+		wxStaticText* st = new wxStaticText( (wxWindow *)parent, -1,
+			label,
+			obj->GetPropertyAsPoint( _("pos") ),
+			obj->GetPropertyAsSize( _("size") ),
+			obj->GetPropertyAsInteger( _("style") ) | obj->GetPropertyAsInteger( _("window_style") ) );
```
Comment (jhasse) [inline plugins/common/common.cpp]: Please don't include the added spaces as it destroys the diff history. Sorry for the trouble and that I didn't mention this earlier :/

**Comment by jhasse** [comment]
Thanks :)

**Pull request merged**